influxdb_iox/tests/end_to_end_cases/management_cli.rs

```diff
@@ -669,6 +669,208 @@ async fn claim_database() {
         )));
 }
 
+#[tokio::test]
+async fn force_claim_database() {
+    let server_fixture = ServerFixture::create_shared(ServerType::Database).await;
+    let addr = server_fixture.grpc_base();
+    let db_name = rand_name();
+    let db = &db_name;
+
+    // Create a database on the server
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("database")
+        .arg("create")
+        .arg(db)
+        .arg("--host")
+        .arg(addr)
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("Created"));
+
+    // Release database returns the UUID
+    let stdout = String::from_utf8(
+        Command::cargo_bin("influxdb_iox")
+            .unwrap()
+            .arg("database")
+            .arg("release")
+            .arg(db)
+            .arg("--host")
+            .arg(addr)
+            .assert()
+            .success()
+            .stdout(predicate::str::contains(format!(
+                "Released database {}",
+                db
+            )))
+            .get_output()
+            .stdout
+            .clone(),
+    )
+    .unwrap();
+    let db_uuid = stdout.lines().last().unwrap().trim();
+
+    // delete the owner file <dir>/dbs/<uuid>/owner.pb
+    let mut owner_file: PathBuf = server_fixture.dir().into();
+    owner_file.push("dbs");
+    owner_file.push(db_uuid);
+    owner_file.push("owner.pb");
+
+    println!("Deleting {:?}", owner_file);
+    Command::new("rm")
+        .arg(owner_file.to_string_lossy().to_string())
+        .assert()
+        .success();
+
+    // Claiming db will now not work (no owner file)
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("database")
+        .arg("claim")
+        .arg(db_uuid)
+        .arg("--host")
+        .arg(addr)
+        .assert()
+        .failure()
+        .stderr(predicate::str::contains("owner.pb not found"));
+
+    // But does work when --force is supplied
+    Command::cargo_bin("influxdb_iox")
+        .unwrap()
+        .arg("database")
+        .arg("claim")
+        .arg(db_uuid)
+        .arg("--host")
+        .arg(addr)
+        .arg("--force")
+        .assert()
+        .success()
+        .stdout(predicate::str::contains(format!(
+            "Claimed database {}",
+            db_name
+        )));
+}
+
+#[tokio::test]
+async fn migrate_database_from_one_server_to_another() {
```

Comment:
Hm, this test name is giving me a bit of pause... I expected it to be testing the normal "release on one server, claim on another" migration path that *doesn't* involve moving files in object storage, but this is testing your use case where you *do* want to move files in object storage for debugging purposes. Maybe...

```suggestion
async fn migrate_database_object_store() {
```

or 

```suggestion
async fn migrate_database_files_from_one_server_to_another() {
```

or 

```suggestion
async fn migrate_database_files_and_force_claim() {
```

?

Reply:
👍  I like `migrate_database_files_from_one_server_to_another` (this is meant to be an end to end test of the workflow I am actually doing -- namely copying an entire database from object store somewhere locally to my machine (aka a different object store) and reimporting it